app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    args = command.split()
+
+    if len(args) != 3 or args[0] != "mv":
+        raise ValueError(
+            "Invalid command format. Use: mv <source> <destination>"
+        )
+
+    source, destination = args[1], args[2]
+
+    if not os.path.isfile(source):
```

Comment:
It might be beneficial to check if the source is a directory and raise an error if so, as this function seems intended for files only.